usaspending_api/references/v1/views.py

```diff
@@ -27,6 +27,7 @@ def post(self, request, format=None):
         m = hashlib.md5()
         m.update(json_req)
         hash = m.hexdigest().encode("utf8")
+        hash = str(hash)[2:-1]
```

Comment:
Could there be a situation
A: The hash string doesn't contain the `b''`? 
B: The hash value is short/empty?

It is probably prudent to make sure the string contained a leading `b'` before trimming